evalml/automl/automl_search.py

```diff
@@ -389,9 +386,14 @@ def search(self, X, y, data_checks="auto", feature_types=None, show_iteration_pl
                 X = pd.DataFrame(X)
             X = ww.DataTable(X)
 
+        text_column_vals = X.select('natural_language')
```

Comment:
This produces a warning if there are no text columns in `X`, open to any suggestions on how to suppress that!

Reply:
Good catch! I'll file that in woodwork. I don't think there should be a warning in this case; we can discuss on that ticket.

For this PR, you can add:
```python
warnings.filterwarnings('ignore', 'The following selectors were not present in your DataTable')
```

Reply:
I think this would this work?

```python
from woodwork.logical_types import NaturalLanguage

 [col_name for col_name, ltype in X.logical_types.items() if ltype == NaturalLanguage]
```

We should probably file an issue with woodwork though.

Reply:
Filed https://github.com/FeatureLabs/woodwork/issues/322 for this warning

Reply:
Yes, that should work. We may fix this due to the issue Dylan [raised](https://github.com/FeatureLabs/woodwork/issues/322).

Reply:
@freddyaboulton oops I had written back but didn't submit my comments until after you commented 😆 

I think we can just suppress the warning, yeah?